src/pybind/mgr/mgr_util.py

```diff
@@ -320,6 +320,23 @@ def format_bytes(n, width, colored=False):
     return format_units(n, width, colored, decimal=False)
 
 
+def to_yaml_or_json(what: Any, format: str) -> str:
```

Comment:
Suggestion:
def to_markup_lang(what: Any, format: str) -> str

Reply:
hm. we're not supporting XML yet.